Pom 465 remove user

cezmc opened the pull request.

Commit:
POM-465 Remove User, Ask popup, Confirmation popup

Commit:
Merge remote-tracking branch 'upstream/develop' into POM-465_remove_user

Commit:
Merge remote-tracking branch 'upstream/develop' into POM-465_remove_user

klimtron commented on src/app/my-account/confirm-remove-account/confirm-remove-account.component.ts:
```diff
@@ -0,0 +1,17 @@
+  styleUrls: ['./confirm-remove-account.component.scss'],
+})
+export class ConfirmRemoveAccountComponent {
+  // eslint-disable-next-line @angular-eslint/no-output-on-prefix
```
Wywalić ten disable i zmienić nazwę"onClosed" na cos w stylu confirm

klimtron commented on src/app/shared/components/remove-account-success/remove-account-success.component.ts:
```diff
@@ -0,0 +1,23 @@
+  styleUrls: ['./remove-account-success.component.scss'],
+})
+export class RemoveAccountSuccessComponent {
+  // eslint-disable-next-line @angular-eslint/no-output-on-prefix
```
To samo co wyżej, zmienić nazwę Outputu z "onClosed"

klimtron commented on src/app/shared/components/remove-account-success/remove-account-success.component.scss:
```diff
@@ -0,0 +1,36 @@
+  margin-bottom: 25px;
+}
+
+::ng-deep .dialog-remove-account-success .mat-dialog-container {
```
ponoć ::ng-deep jest deprecated

neu5 replied:
https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

Commit:
POM-465 Remove User, Ask popup, Confirmation popup
- remove unused code comments
- change Output name
- global styles for success dialog
